### DISCUSSION TITLE
AppGallery or standalone release for Huawei devices

### DISCUSSION THREAD

**Discussion by OzancanKaratas** [BearWare/TeamTalk5#2006] [Q&A]
### Description

The Huawei devices doesn't have a TeamTalk client. Can you provide a standalone file or AppGallery release?

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
No you can only use Google Play.

**Comment by schulle4u** [comment]
TeamTalk also runs smoothly on Android devices where there is no Play Store or Play Services available. I have already successfully installed the client on an Amazon Fire tablet using a manually downloaded APK file. Google still serves most of the Android ecosystem, but it would be unfair to continue excluding the growing user base of alternative Android manufacturers. So +1 for a standalone release.

**Comment by amirmahdifard** [comment]
@bear101 do you agree that from now on up on releaces we also releace the universal apk file? it will be conpatible with all android devices just that single file, and nearly all apps are doing that. so everyone can just download the file, share it freely, with out needing play store. They can use both.
